### ISSUE TITLE
vmalert - static assets fail to load when http.pathPrefix is specified

### ISSUE THREAD

**Issue by krakazyabra** [VictoriaMetrics/VictoriaMetrics#4349]
### Describe the bug

`--http.pathPrefix` option breaks webGUI of vmalert

### To Reproduce

deploy vmalert (helm-chart) with
```jsonnet
extraArgs: {
  'http.pathPrefix': '/vmalert',
}
ingress+: {
  hosts: [
    {
      name: "victoriametrics.domain.org",
      path: "/vmalert",
    },
  ],
}
```

### Version

```shell
/ # /vmalert-prod --version
vmalert-20230518-212228-tags-v1.91.0-0-g73dfb030d
```

### Logs

```shell
{"ts":"2023-05-26T14:11:20.911Z","level":"warn","caller":"VictoriaMetrics/app/vmalert/web.go:147","msg":"remoteAddr: \"10.113.134.91:57928\", X-Forwarded-For: \"10.31.1.173\"; requestURI: /vmalert/static/css/bootstrap.min.css; unsupported path requested: \"/static/css/bootstrap.min.css\" "}
```

### Used command-line flags

--http.pathPrefix=/vmalert

**Comment by hagen1778** [comment]
Thanks for report!
@krakazyabra could you please try if https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4351/commits/abe355ffd49325d6f9ca796ae6aadae085325ec6 helps with the issue? See how to build vmalert from sources [here](https://docs.victoriametrics.com/vmalert.html#how-to-build-from-sources).

**Comment by krakazyabra** [comment]
@hagen1778 seems that fix works. 
Also I fount one more bug connected to WebUI: on alerts tab, when there are active alerts, near alert name there is link `source`. this link point to pod's hostname instead of ingress. 

![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/26598203/32a6d804-7527-4313-b1cd-7eee72ee5a7c)

**Comment by krakazyabra** [comment]
@hagen1778 Thanks, now it's better!

**Comment by hagen1778** [comment]
> Also I fount one more bug connected to WebUI: on alerts tab, when there are active alerts, near alert name there is link source. this link point to pod's hostname instead of ingress.

The URL here is controlled by cmd-line flag `-external.url`:
```
  -external.url string
     External URL is used as alert's source for sent alerts to the notifier. By default, hostname is used as address.
```
@k1rk @zekker6 @f41gh7 does it make sense to you setting this flag value with ingress address by default in helm chart?

**Comment by hagen1778** [comment]
Hello @krakazyabra! The fix was included in https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.91.1

**Comment by hagen1778** [comment]
@krakazyabra https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.91.1 contains a bug. Please use https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.91.2 instead.

**Comment by valyala** [comment]
FYI, the bugfix has been also included in [long-term support release v1.79.14](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.79.14).

**Comment by valyala** [comment]
The bugfix has been also included in the [long-rem support release v1.87.7](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.87.7).
